utils/makegqlrequest.ts

```diff
@@ -0,0 +1,31 @@
+import { BE_DEPLOYMENT_DOMAIN } from "./secrets";
+
+interface GraphqlResponse {
+  data: any;
```

Comment:
can you disable unexpected-any for this file? otherwise we'll keep getting lint errors on subsequent PRs due to this file